tests/system/source/main.d

```diff
@@ -98,6 +98,8 @@ int main (string[] args)
                     const blocks = client.getBlocksFrom(height, 10);
                     writefln("%s %s has blocks:\n%s", PREFIX, nodeFromClientIndex(idx), prettify(blocks));
                     writefln("%s ----------------------------------------", PREFIX);
+                    assert(blocks[$ -1].txs.length > 0,
+                        format!"%s %s has block at height %s with no transactions!"(PREFIX, nodeFromClientIndex(idx), node_height));
```

Comment:
I think we need to be a bit smarter, as it could spuriously fails if it sends the tx during nomination.

Reply:
You mean like test if we have several empty blocks in a row?

Reply:
Yes, just like we do in the network tests: https://github.com/bosagora/agora/blob/f29e458fb6e80f9581650db7f44f84e945c62469/source/agora/test/Base.d#L1172-L1192

Reply:
Updated